MediaAdminBundle/Resources/views/Tree/tree.html.twig

```diff
@@ -0,0 +1,10 @@
+{% import "OpenOrchestraMediaAdminBundle:Tree:tree-macros.html.twig" as macros %}
```

Comment:
Maybe you speak about tree.html.twig. I duplicate this file in order to controller of mediaAdmin calling twig in mediaAdmin. @nicolasThal